scout/adapter/mongo/query.py

```diff
@@ -179,6 +180,34 @@ def build_query(self, case_id, query=None, variant_ids=None, category="snv"):
                 else:  # sv
                     coordinate_query = [self.sv_coordinate_query(query)]
 
+            elif criterion == "chrom_pos" and query.get("chrom_pos"):  # filter by coordinates
```

Comment:
I've been testing the branch on a local instance and I see that every time a user inserts coordinates in the chrom_pos fields -> chromosome, start, and stop fields are also updated (not applying the eventual padding). If you apply the padding in the web page (at the same time you update these fields) then you don't have to touch this file at all, because you can perform the search using those values instead (only chromosome, start and stop). It would avoid a lot of additional code in a function that is already exploding (not your fault, it's already a monster function as it is)